docs/release-info/release-notes.md

```diff
@@ -57,46 +57,543 @@ For tips about how to write a good release note, see [Release notes](https://git
+- Improved memory management by dividing the amount of `partitionStatsMemory` by two to account for two simultaneous statistics collectors [#17216](https://github.com/apache/druid/pull/17216)
+- Fixed NPE in `CompactSegments` [#16713](https://github.com/apache/druid/pull/16713)
+- Fixed Parquet reader to ensure that Druid reads the required columns for a filter from the Parquet data files [#16874](https://github.com/apache/druid/pull/16874)
+- Fixed a distinct sketches issue where Druid called `retainedKeys.firstKey()` twice when adding another sketch [#17184](https://github.com/apache/druid/pull/17184)
+- Fixed a `WindowOperatorQueryFrameProcessor` issue where larger queries could reach the frame writer's capacity preventing it from outputting all of the result rows [#17209](https://github.com/apache/druid/pull/17209)
+- Fixed native ingestion task failures during rolling upgrades from a version before Druid 30 [#17219](https://github.com/apache/druid/pull/17219)
+
+### SQL-based ingestion
+
+#### Optimized S3 storage writing for MSQ durable storage
+
+For queries that use the MSQ task engine and write their output to S3 as durable storage, uploading chunks of data is now faster.
+
+[#16481](https://github.com/apache/druid/pull/16481)
 
-##### Other SQL-based ingestion improvements
+#### Improved lookup performance
 
-#### Streaming ingestion
+Improved lookup performance for queries that use the MSQ task engine by only loading required lookups. This applies to both ingestion and querying.
 
-##### Other streaming ingestion improvements
+[#16358](https://github.com/apache/druid/pull/16358)
+
+#### Other SQL-based ingestion improvements
+
+- Added the ability to use `useConcurrentLocks` in task context to determine task lock type [#17193](https://github.com/apache/druid/pull/17193)
+- Reduced memory usage when transferring sketches between the MSQ task engine controller and worker [#16269](https://github.com/apache/druid/pull/16269)
+- Improved error handling when retrieving Avro schemas from registry [#16684](https://github.com/apache/druid/pull/16684)
+- Fixed issues related to partitioning boundaries in the MSQ task engine's window functions [#16729](https://github.com/apache/druid/pull/16729)
+- Fixed a boost column issue causing quantile sketches to incorrectly estimate the number of output partitions to create [#17141](https://github.com/apache/druid/pull/17141)
+- Fixed an issue with `ScanQueryFrameProcessor` cursor build not adjusting intervals [#17168](https://github.com/apache/druid/pull/17168)
+- Improved worker cancellation for the MSQ task engine to prevent race conditions [#17046](https://github.com/apache/druid/pull/17046)
+- Improved memory management to better support multi-threaded workers [#17057](https://github.com/apache/druid/pull/17057)
+- Reduced memory usage when transferring sketches between the MSQ task engine controller and worker [#16269](https://github.com/apache/druid/pull/16269)
```

Comment:
This is duplicated from mentioned from line 245. Also, a better way to word it might be "Add new serialization of sketches between MSQ controller and worker to reduce memory usage".